### PR TITLE
Florian/new samples

### PR DESCRIPTION
**Pull request by frivoal**

### CODE REVIEW

**Comment by kwkbtr** [comment]
Dashes(?) in Gutenberg samples seem not to be rendered correctly.
E.g. https://github.com/vivliostyle/vivliostyle.js/blob/010c67191bab892d1023d46912a5f34d61e03b6d/samples/gutenberg/Alice.html#L200

(screenshot with Safari)

**Comment by frivoal** [comment]
Strange, the dash problem does not reproduce here. Anyway, I've replaced the codepoint by an html entity, so all should be good now.

File: samples/scholarly/scholarly.css [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+code {
+  overflow: visible;
+  white-space: pre-wrap;
+}
+pre {
+  border: solid 1px #DFDFDF;
+  border-left: solid #358CCB 10px;
+  max-width: 100%;
+  box-sizing: border-box;
+  margin:0;
```
Comment (kwkbtr) [inline samples/scholarly/scholarly.css:278]: I think adding some `padding-left` here would make code blocks look nicer.

Comment (frivoal) [inline samples/scholarly/scholarly.css:278]: You mean in the pre { } block? yes, probably, although not too much as we have fairly long lines and not very wide boxes for pre, so I'd prefer not to wrap lines too much.

How about 1em?

Comment (kwkbtr) [inline samples/scholarly/scholarly.css:278]: Yes, 1em would be fine.

**Comment by kwkbtr** [comment]
The new samples have been added by #166 (Ishida-san adjusted some styles).

**Pull request closed without merging**